bin/prepare_project_edition.sh

```diff
@@ -62,8 +77,25 @@ git init; git add . > /dev/null;
 # Execute Ibexa recipes
 docker exec install_dependencies composer recipes:install ibexa/${PROJECT_EDITION} --force
 
+# Add other dependencies if required
```

Comment:
This part is handling the `dependencies.json` file - we're adding the specified dependencies to the project.